shared/actions/settings.js

```diff
@@ -371,11 +382,17 @@ function* settingsSaga(): Saga.SagaGenerator<any, any> {
   yield Saga.safeTakeLatest(SettingsGen.notificationsToggle, _toggleNotificationsSaga)
   yield Saga.safeTakeLatestPure(SettingsGen.dbNuke, _dbNukeSaga)
   yield Saga.safeTakeLatestPure(SettingsGen.deleteAccountForever, _deleteAccountForeverSaga)
-  yield Saga.safeTakeLatestPure(SettingsGen.loadSettings, _loadSettings, _loadSettingsSuccess)
+  yield Saga.safeTakeEveryPure(SettingsGen.loadSettings, _loadSettings, _loadSettingsSuccess)
```

Comment:
Shouldn't latest be ok? what is the rule of thumb again?

Reply:
Yep, both seem okay, I took from yesterday's call that we should generally prefer Every unless there's a good reason to allow cancellation and we can reason about what state things would be left in, and I didn't feel like those were true so switched it out while I was in here.  No big deal either way though.

Reply:
i think 99% of the time we was every. we can handle last type stuff more explicitly with bookkeeping and ignoring or canceling stuff (usually ignoring)